tests/ccm.tests.ps1

```diff
@@ -0,0 +1,75 @@
+[CmdletBinding()]
+Param(
+    [Parameter(Mandatory)]
+    [String]
+    $Fqdn
+)
+
+Describe "Chocolatey Central Management Configuration" {
+    Context "Services" {
+        BeforeAll {
+            function Get-RemoteCertificate {
```

Comment:
Is this function the same in both places? If it is, we should pull it out of the tests here and just load it into scope before the tests (inside `Start-C4bVerification` we could either dot-source a script with this function in it, or we could include this. Alternatively we could put this in it's own file and dot-source it to get it into scope...